src/Error/Debugger.php

```diff
@@ -401,26 +401,28 @@ public static function formatTrace(Throwable|array $backtrace, array $options =
             if (in_array($signature, $options['exclude'], true)) {
                 continue;
             }
-            if ($options['format'] === 'shortPoints') {
+
+            $format = $options['format'];
+            if ($format === 'shortPoints') {
                 $back[] = [
                     'file' => self::trimPath($frame['file']),
                     'line' => $frame['line'],
                     'reference' => $reference,
                 ];
-            } elseif ($options['format'] === 'points') {
+            } elseif ($format === 'points') {
                 $back[] = ['file' => $frame['file'], 'line' => $frame['line'], 'reference' => $reference];
-            } elseif ($options['format'] === 'array') {
+            } elseif ($format === 'array') {
                 if (!$options['args']) {
                     unset($frame['args']);
                 }
                 $back[] = $frame;
-            } elseif ($options['format'] === 'text') {
+            } elseif ($format === 'text') {
                 $path = static::trimPath($frame['file']);
                 $back[] = sprintf('%s - %s, line %d', $reference, $path, $frame['line']);
             } else {
                 debug($options);
```

Comment:
@markstory I am guessing this one is not supposed to be here? Seems like no one noticed for like 2+ years.

Reply:
lol. Yeah this can be removed.